cypress/integration/rancid_app_spec.js

```diff
@@ -132,13 +132,11 @@ describe('Radioactive Toothbrushes', () => {
       cy.visit(baseURL);
     });
 
-    // it.only('Should have a loading message', () => {
-    //   cy.get('section > a')
-    //     .contains('Mulan')
-    //     .click()
-    //     // .get('h2').should('be.visible');
-    //     .get('.message').contains('Looking for your movie...');
-    // });
+    it('Should have a loading message when the film\'s url is visited', () => {
```

Comment:
This work around is good. I'm glad we're able to test that there's some type of loading message.